packages/web-twig/src/Resources/components/Breadcrumbs/BreadcrumbsItem.twig

```diff
@@ -0,0 +1,42 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _href = props.href -%}
+{%- set _isCurrent = props.isCurrent | default(false) -%}
+{%- set _isGoBackOnly = props.isGoBackOnly | default(false) -%}
+{%- set _unsafeIconStart = props.UNSAFE_iconStart | default(null) -%}
+{%- set _unsafeIconEnd = props.UNSAFE_iconEnd | default(null) -%}
+{%- set _children = block('content') -%}
+
+{# Class names #}
+{%- set _displayNoneClassName = _spiritClassPrefix ~ 'd-none' -%}
+{%- set _displayTabletNoneClassName = _spiritClassPrefix ~ 'd-tablet-none' -%}
+{%- set _displayTabletFlexClassName = _spiritClassPrefix ~ 'd-tablet-flex' -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{% if _isGoBackOnly is not same as(true) %}
+    {%- set _classNames = [ _styleProps.className, _displayNoneClassName, _displayTabletFlexClassName ] -%}
+{% else %}
+    {%- set _classNames = [ _styleProps.className, _displayTabletNoneClassName ] -%}
```

Comment:
so now we lose the option to show goback on any BP we want, right?

Reply:
I think I do not understand...

Reply:
I still do not understand how you want to achieve this modification you are writing about. I think the functionality is already baked in the same way as it was before. E.g. if you pass down the `items` prop the `BreadcrumbsItem` will be used. Or. You can pass down the children and fabricate the breadcrumbs your way. Whether you use the `BreadcrumbsItem` to help or just plain HTML. So you can modify this as you wish.